src/runtime/security/replica_access_controller.h

```diff
@@ -27,10 +27,12 @@ class replica_access_controller : public access_controller
 public:
     replica_access_controller(const std::string &name);
     bool allowed(message_ex *msg);
+    void reset(const std::string &users);
```

Comment:
I think `update` or other name may be better, `reset` sounds like reset to default value.